traffic_ops/install/bin/input.json

```diff
@@ -67,5 +67,102 @@
     },
     {
       "LDAP Search Base":"",
-      "input.json"      169      L,
-      4044      C
\ No newline at end of file
+      "config_var":"search_base"
+    }
+  ],
+  "/opt/traffic_ops/install/data/json/users.json":[
+    {
+      "Administration username for Traffic Ops":"root",
+      "config_var":"tmAdminUser"
+    },
+    {
+      "Password for the admin user":"default",
+      "config_var":"tmAdminPw",
+      "hidden":"1"
+    }
+  ],
+  "/opt/traffic_ops/install/data/profiles/":[
+
+  ],
+  "/opt/traffic_ops/install/data/json/openssl_configuration.json":[
+    {
+      "Do you want to generate a certificate?":"yes",
+      "config_var":"genCert"
+    },
+    {
+      "Country Name (2 letter code)":"XX",
+      "config_var":"country"
+    },
+    {
+      "State or Province Name (full name)":"Default State",
+      "config_var":"state"
+    },
+    {
+      "Locality Name (eg, city)":"Default City",
+      "config_var":"locality"
+    },
+    {
+      "Organization Name (eg, company)":"Default Company Ltd",
+      "config_var":"company"
+    },
+    {
+      "Organizational Unit Name (eg, section)":"",
+      "config_var":"org_unit"
+    },
+    {
+      "Common Name (eg, your name or your server's hostname)":"example.com",
+      "config_var":"common_name"
+    },
+    {
+      "RSA Passphrase":"password",
```

Comment:
Passwords in files always worry me, we should maybe evaluate a command line option or have a separate file that can be stored locally to be safe.